lib/commands/update.js

```diff
@@ -128,6 +141,66 @@ class UpdateCommand extends Command {
         return yarnInstall(ctx.ui, ctx.zip);
     }
 
+    majorSupport(ctx) {
+        const majorSupport = require('../utils/major-support');
+        let gscanReport;
+        let demoPost;
+
+        return majorSupport({
+            dir: ctx.instance.dir,
+            database: ctx.instance.config.get('database')
+        }).then((response) => {
+            gscanReport = response.gscanReport;
+            demoPost = response.demoPost;
+
+            this.ui.log(`\nTheme score: ${gscanReport.results.score.value}\n`);
+
+            if (gscanReport.results.error.length) {
+                this.ui.log('### Errors\n');
+                gscanReport.results.error.forEach((error) => {
+                    this.ui.log(`Rule: ${error.rule}`);
+                    this.ui.log(`File: ${map(error.failures, 'ref').join(',')}`);
+                });
+            }
+
+            if (gscanReport.results.warning.length) {
+                this.ui.log('### Warnings\n');
+                gscanReport.results.warning.forEach((warning) => {
+                    this.ui.log(`Rule: ${warning.rule}`);
+                    this.ui.log(`File: ${map(warning.failures, 'ref').join(',')}`);
+                });
+            }
+
+            if (!gscanReport.results.error.length && !gscanReport.results.warning.length) {
+                this.ui.log('Theme is compatible.\n')
+            }
+
+            return this.ui.prompt({
```

Comment:
You should be able to use `this.ui.confirm` here :)

Reply:
👍